content/pop-shop.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Pop!_Shop Basics
+description: >
+  Learn how to use the Pop!_Shop to install software on your Pop!_OS computer.
+keywords:
+    - Pop!_Shop
+    - Pop!_OS
+    - Pop!_OS Beginner
+    - System76
+    - Software
+    - Install
+    - Install Software
+    - Install Applications
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: pop
+tableOfContents: true
+---
+
+Welcome to Pop!\_Shop - the software center for Pop!\_OS. Pop!\_Shop is a user-friendly graphical interface for installing and removing software on your computer. Pop!\_Shop is based on Elementary AppCenter
+
+## Launching Pop!_Shop
+
+To launch Pop!_Shop, click the `Pop!_Shop` icon in the dock, or search for it after pressing the `Super` key. 
+
+![Pop!_Shop](/images/pop-shop/pop-shop.png)
+
+## Searching for Software
+
+To search for software, type the name of the software you are looking for in the search bar at the top of the window.
+
+![Pop!_Shop](/images/pop-shop/pop-shop-search.png)
+
+## Installing Software
```

Comment:
How much value is there in listing the buttons of the application in an article? These sentences aren't really adding anything beyond the name of the button.